cli/src/collector/export.rs

```diff
@@ -40,7 +40,7 @@ pub(crate) fn export_specs(config: &AppConfig, fetcher: impl Fetcher) -> Result<
             ExportChainSpec {
                 title: chain.title.as_ref().unwrap_or(&chain.name).clone(),
                 color: chain.color.clone(),
-                rpc_endpoint: chain.rpc_endpoint.clone(),
+                rpc_endpoint: chain.rpc_endpoints[0].clone(), // keep only the first one
```

Comment:
Is it possible to get the node that was used for getting the chainspec?

Reply:
It is possible, although I'm not sure how useful that is. 
I see having multiple endpoints as a failover option. That means there is an "official" endpoint and some others for backup\failover\load balancing. Not sure if users are interested in knowing the details. 
Moreover, chainspecs are fetched with every `make collector`, so we might get different endpoints after each deployment. That might be confusing.

Reply:
Got it. It's a very good point!